MAUI/Scheduler/timeline-views.md

```diff
@@ -43,6 +43,8 @@ this.Scheduler.TimelineView.NumberOfVisibleDays = 3;
 {% endhighlight %}
 {% endtabs %}
 
+![visible-days-count-in-maui-scheduler](images/timeline-views/number-of-visibledays-in-maui-scheduler.png)
+
```

Comment:
Timeline view image not represent days count, you can customize start, hour cour to see number of days in single view